test/test_graph/test_graph_cbd.py

```diff
@@ -109,3 +122,21 @@ def testCbdReified(get_graph):
     )
 
     assert len(g.cbd(EX.R6)) == (3 + 5 + 5), "cbd() for R6 should return 12 triples"
+
+
+def testCbdExample():
```

Comment:
```suggestion
def test_cbd_example():
```

Function names should conform to PEP8[[ref](https://github.com/RDFLib/rdflib/blob/master/docs/developers.rst#introduction)]:

https://peps.python.org/pep-0008/#function-and-variable-names

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.